astrodb_utils/sources.py

```diff
@@ -211,10 +211,36 @@ def coords_from_simbad(source):
 
     return simbad_skycoord
 
+def simbad_name_resolvable(source, ra, dec):
```

Comment:
add doc string. especially include the Returns section to describe what it returns.